src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -748,20 +753,70 @@ const UsersConfigurationRow = ({
     );
 };
 
+const SshKeysRow = ({
+    id, item, onChange, idx, removeitem,
+}) => {
+    const [isLoading, setIsLoading] = useState(false);
+
+    const handleClear = (_event) => {
+        onChange(idx, "file", "");
+        onChange(idx, "value", "");
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20230814-091115-8af58863-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)